app/code/Magento/Customer/Controller/Plugin/Account.php

```diff
@@ -3,72 +3,78 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
+
 namespace Magento\Customer\Controller\Plugin;
 
+use Closure;
+use Magento\Customer\Controller\AccountInterface;
 use Magento\Customer\Model\Session;
-use Magento\Framework\App\ActionInterface;
 use Magento\Framework\App\RequestInterface;
 use Magento\Framework\App\ResponseInterface;
-use Magento\Framework\App\Action\AbstractAction;
 use Magento\Framework\Controller\ResultInterface;
 
+/**
+ * Plugin verifies permissions using Action Name against injected (`fontend/di.xml`) rules
+ */
 class Account
 {
     /**
      * @var Session
      */
-    protected $session;
+    private $session;
```

Comment:
I would also keep the scope of visibility as it's.

Reply:
Are you going to inherit plugin? o.O

Reply:
I hope that's obvious, I'm not :)
However, I prefer to avoid making exceptions for edge cases.
If I'm not mistaken, changing the scope of visibility of method/property was always prohibited, as well as injecting constructor param in a backward-incompatible way, even for non `@api` classes.